docs/pages/blog/you-might-not-need-typescript-project-references.mdx

```diff
@@ -67,6 +68,8 @@ const nextConfig = {
 module.exports = nextConfig;
 ```
 
+<Callout type="info">As of Next.js 13.1, you no longer need the `next-transpile-modules` package. For more information, visit the Next.js blog post [here](https://nextjs.org/blog/next-13-1#built-in-module-transpilation-stable).</Callout>
```

Comment:
Personal preference here so feel free to take it or leave it, but I generally prefer to avoid linking with "here". Instead, using something that informs the user about where they are going.

```suggestion
<Callout type="info">As of Next.js 13.1, you no longer need the `next-transpile-modules` package. For more information, visit the Next.js [Built-in module transpilation](https://nextjs.org/blog/next-13-1#built-in-module-transpilation-stable) blog post.</Callout>
```